Exec/gravity_tests/uniform_cube_sphere/inputs

```diff
@@ -15,6 +15,11 @@ amr.n_error_buf      = 0 0 0 0 0 0 0 0 0 0 0
 amr.blocking_factor  = 8
 amr.max_grid_size    = 8
 
+amr.refinement_indicators = denerr
+
+amr.refine.denerr.value_greater = 1.0e0
+amr.refine.denerr.field_name = density
+
```

Comment:
don't you need a max level here?

Reply:
No. By default the max level is 1000 so we only need to explicitly set the max level if we want the AMR not to be applied on every level.